python/py-qdarkstyle/Portfile

```diff
@@ -24,13 +24,16 @@ checksums           sha256  6a967c4b664446f8bed9df12d1032cf68cb54f186bfc9cbfdbbc
                     rmd160  95b7cbcdbec958e2ee685aff8348e5601a2a1c35 \
                     size    405579
 
-python.versions     27 35 36 37
+python.versions     27 35 36 37 38
```

Comment:
upstream does not declare compatibility with Python 3.8; I assume you verified it is in fact working correctly?

Reply:
I was able to successfully run the [example script](https://github.com/ColinDuquesnoy/QDarkStyleSheet/blob/2dc4ab52a70c9ebd66f1e0afa1a551a73565db81/example/example.py).

Reply:
alright, that's should do it - thanks!